bindings/csharp/Ckzg.Test/ReferenceTests.cs

```diff
@@ -354,4 +360,158 @@ public void TestVerifyBlobKzgProofBatch()
     }
 
     #endregion
+
+    #region ComputeCellsAndKzgProofs
+
+    private class ComputeCellsAndKzgProofsInput
+    {
+        public string Blob { get; set; } = null!;
+    }
+
+    private class ComputeCellsAndKzgProofsTest
+    {
+        public ComputeCellsAndKzgProofsInput Input { get; set; } = null!;
+        public List<List<string>>? Output { get; set; } = null!;
+    }
+
+    [TestCase]
+    public void TestComputeCellsAndKzgProofs()
+    {
+        Matcher matcher = new();
+        matcher.AddIncludePatterns(new[] { "*/*/data.yaml" });
+
+        IEnumerable<string> testFiles = matcher.GetResultsInFullPath(_computeCellsAndKzgProofsTests);
+        Assert.That(testFiles.Count(), Is.GreaterThan(0));
+
+        foreach (string testFile in testFiles)
+        {
+            string yaml = File.ReadAllText(testFile);
+            ComputeCellsAndKzgProofsTest test = _deserializer.Deserialize<ComputeCellsAndKzgProofsTest>(yaml);
+            Assert.That(test, Is.Not.EqualTo(null));
+
+            byte[] cells = new byte[CellsPerExtBlob * Ckzg.BytesPerCell];
+            byte[] proofs = new byte[CellsPerExtBlob * Ckzg.BytesPerProof];
+            byte[] blob = GetBytes(test.Input.Blob);
+
+            try
+            {
+                Ckzg.ComputeCellsAndKzgProofs(cells, proofs, blob, _ts);
+                Assert.That(test.Output, Is.Not.EqualTo(null));
+                byte[] expectedCells = GetFlatBytes(test.Output.ElementAt(0));
```

Comment:
Can the type of it be an array of arrays instead? Could you access elements just by [index]?

Reply:
Hmm I'm not sure about this one. Let's look into this afterwards.